helm/tailing-sidecar-operator/values.yaml

```
@@ -14,6 +14,13 @@ operator:
     requests:
       cpu: 100m
       memory: 20Mi
+  livenessProbe: {}
+    # tcpSocket:
+    #   port: 12800
+    # initialDelaySeconds: 15
```

it would be good to have default setting visible here, so please change to:
```suggestion
    # initialDelaySeconds: 1
```